requirements/requirements_prod.txt

```diff
@@ -6,3 +6,5 @@ django-crispy-forms==1.11.2
 django-dotenv==1.4.2
 psycopg2-binary==2.9.3
 gunicorn==20.1.0
+factory-boy==3.2.1
```

Comment:
Ideally, when you are adding a package that don't belong to a current PR, submit it in a separate commit saying that you forgot to include it in the previous PR. But next time. Let's merge this PR as it is 👍